internal/contentdata/repository.go

```diff
@@ -71,6 +72,9 @@ func (r *Repository) CreateTable(ctx context.Context, tx *connection.Tx, table *
 	}
 	query := fmt.Sprintf("CREATE TABLE `%s` (%s)", ref.TableId, strings.Join(fields, ","))
 	if _, err := tx.Tx().ExecContext(ctx, query); err != nil {
+		if strings.HasSuffix(err.Error(), "already exists") {
```

Comment:
I can't find error instance when table duplicates. So I check as suh. If any, please tell me more smart way.

Reply:
I think that by running `CreateTable` after `r.dataset.AddTable` , there is no need to determine if it is a duplicated table or not, so this difference will also be unnecessary.

Reply:
I see. Thnaks!